### ISSUE TITLE
Show/Hide mini details pane on icon hover

### ISSUE THREAD

**Issue by rdev32** [BartoszCichecki/LenovoLegionToolkit#1027]
### Rules

- [ ] I made myself familiar with the README.
- [ ] I checked closed issues for similar ones and did not find a similar feature request mentioned before.
- [ ] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.16.2

### OS

Windows 11 23H2

### Device

Legion 5 Pro 16ACH6H

### Is your feature request related to a problem?

I want to request a toggle for hiding this 
<img width="318" alt="image" src="https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/78289548/68969fd2-9c5b-485e-923f-ebf7ec73f37b">

### How would you like the problem to be solved?

By creating an option that lets you hide that mini status window

### What alternatives have you considered?

By donating and asking on the github page for that feature

**Comment by BartoszCichecki** [comment]
If you read the readme you will find a switch for hiding this tooltip that you can use.

**Comment by rdev32** [comment]
hi I read again the README and the only thing I found related to my suggestion is this piece
> Why don't I see the custom tooltip when I hover LLT icon in tray?
In Windows 10 and 11, Microsoft did plenty of changes to the tray, breaking a lot of things on the way. As a results custom tooltips not always work properly. Solution? Update your Windows and keep fingers crossed.

This is not what I meant
What I meant was to disable that popup that shows info when hovering the LTT tray icon
I was looking for the option as you mentions but unfortunately I found none
